source/draft_auth.rst

```diff
@@ -5,15 +5,14 @@ Authorization
 =============
 
 TimeSync allows for various forms of authentication determined by the
-implementation. The general scheme an authentication module follows looks like
-this:
+implementation. The general scheme of an authentication module follows looks
```

Comment:
Choose either "follows", "looks like this", or add a conjunction between the two.